includes/functions/llms-functions-templates-shared.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Shared template functions
+ *
+ * A "shared" function is any function used by more than one post type.
+ *
+ * @package LifterLMS/Functions
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+if ( ! function_exists( 'llms_template_instructors' ) ) {
+
+	/**
+	 * Get single post instructors template
+	 *
+	 * Used by courses and membership.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	function llms_template_instructors() {
+
+		$post = llms_get_post( get_the_ID() );
+		if ( ! $post || ! $post instanceof LLMS_Post_Model ) {
```

Comment:
```suggestion
		if ( ! $llms_post || ! $llms_post instanceof LLMS_Interface_Post_Instructors ) {
```
I think a check on if `instanceof LLMS_Interface_Post_Instructors` fits better here.
That interface is implemented by `LLMS_Course` and `LLMS_Membership` which have to implement `get_instructors()` method. What do you think?

Reply:
Good point, but we use methods from `LLMS_Post_Model` in the template so I think we should be checking for both.